blazor/pdfviewer/getting-started/web-assembly-application.md

```diff
@@ -163,3 +165,43 @@ Press <kbd>Ctrl</kbd>+<kbd>F5</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>F5</kbd> (m
 ![Blazor PDF Viewer Component](GettingStarted_images/blazor-pdfviewer.png)
 
 > [View Sample in GitHub](https://github.com/SyncfusionExamples/Blazor-Getting-Started-Examples/tree/main/PDFViewer).
+
+## How to load desired PDF for initial loading and change document path at runtime
+
+You can load your own PDF document for initial loading as well as change document at run-time in PDF Viewer WebAssembly projects. To achieve that, you need to create the webservice and add your documents in that webservice project. Then mention that webservice localhost path as service url in your webassembly project.
```

Comment:
can you please modify like below?

**You can load your PDF document for initial loading as well as change the document at run-time in PDF Viewer WebAssembly projects. To achieve that, you need to create the web services and add your documents to that web service project. Then mention that web service localhost path as the service URL in your web assembly project**

Reply:
modified as per your comment.